### PR TITLE
PR: Add silvershiny as a contributor

### PR DESCRIPTION
**Pull request by silvershiny**

### CODE REVIEW

**Review by creative2113 — COMMENTED** [review]
I see.

**Review by shinevue — APPROVED** [review]
Welcome

**Review by creative2113 — APPROVED** [review]
ok

**Comment by BlackGhost2693** [comment]
Good

File: README.md [hunk 1]
```diff
@@ -68,3 +68,4 @@
 #### [Grassman](https://github.com/grasshousedev)
 #### [Storm](https://github.com/felipedev418)
 #### [Shiny7star](https://github.com/shiny7star)
+#### [Silver](https://github.com/silvershiny)
```
Comment (BlackGhost2693) [inline README.md:71]: #### [Blackghost](https://github.com/blackghost2693)

**Review by creative2113 — APPROVED** [review]
go ahead

**Review by techietrend — APPROVED** [review]
Good

**Review by popstar7 — APPROVED** [review]
Welcome

**Review by shiny7star — APPROVED** [review]
Funny

**Review by goldsunshines — APPROVED** [review]
Hi, pretty
Very good

**Review by appleseed619 — APPROVED** [review]
Ok

**Pull request merged**